presto-main/src/test/java/com/facebook/presto/operator/aggregation/reservoirsample/TestWeightedDoubleReservoirSample.java

```diff
@@ -34,9 +34,10 @@ public void testIllegalMaxSamples()
     @Test
     public void testGetters()
     {
-        WeightedDoubleReservoirSample sample = new WeightedDoubleReservoirSample(200);
+        WeightedDoubleReservoirSample reservoir = new WeightedDoubleReservoirSample(200);
```

Comment:
It seems more precise to call it a reservoir (of samples), than a (single) sample. Before this, there was code that seemed strange in the tests: there was a reservoir called a "sample", and then another data structure holding the expected samples, called "samples". It would stand to reason, therefore, that "sample" would hold a single something of which "samples" would hold several, but that was not the case. Currently, both "reservoir" and "samples" hold one or more "sample", which makes sense (to me).